### DISCUSSION TITLE
Reusing Assets in different pipelines

### DISCUSSION THREAD

**Discussion by JSteinbauer** [dagster-io/dagster#11799] [Q&A] [answered]
Dear Dagster Team,

thanks for your amazing tools!

I have a question regarding the reusability of (source) assets in different data pipelines - is that possible? 
Here's an example of what I mean:
![Screenshot from 2023-01-19 17-06-12](https://user-images.githubusercontent.com/62147690/213493291-549e6352-fbc6-42a2-b057-e99271bc6a93.png)
In the graph above, the asset "opp_contacts" is used by the assets "labeled_data" and "labeled_data2". 
Ideally, I'd like to separate the downstream assets into two different groups, such that they are not visualized together (which is very confusing, since the two pipelines are independent from each other). 
Is there any way of which you could think of to achieve this?

Thanks in advance and all the best,
Jakob

**Comment by dpeng817** [comment, marked as answer]
Have you checked out the guide on [asset groups](https://docs.dagster.io/concepts/assets/software-defined-assets#grouping-assets)? Could use that to group the downstream assets separately. Keep in mind that an asset can only be in one asset group though, so you wouldn't want to group `opp_contacts` into one of the downstream asset groups.
